android-patches/scripts/CopyToRepo.bat

```diff
@@ -1,27 +1,27 @@
-@if "%DEBUG%" == "" @echo off

```

Comment:
unintentional change?

Reply:
I mentioned this in another comment on the review. I think VSCode on Mac didn't like the current line endings. I can just fix them up now since they're already on my machine